platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/TeamMember.java

```diff
@@ -9,7 +10,7 @@
         @AssociationOverride(name = "key.user", joinColumns = {@JoinColumn(name = "user_id")}),
         @AssociationOverride(name = "key.team", joinColumns = {@JoinColumn(name = "team_id")})
 })
-public class TeamMember {
+public class TeamMember implements Serializable {
```

Comment:
Class now implements `Serializable` but is missing field `serialVersionUID` which is strongly recommended.